src/hotspot/share/prims/jvmtiExport.cpp

```diff
@@ -1964,6 +1965,7 @@ void JvmtiExport::post_exception_throw(JavaThread *thread, Method* method, addre
   HandleMark hm(thread);
   methodHandle mh(thread, method);
   Handle exception_handle(thread, exception);
+  KeepStackGCProcessedMark ksgcpm(thread);
 
```

Comment:
Nit: It'd be nice to place a small comment about why it is needed.

Reply:
+1 I've never even heard of this thing let alone understand when I would need to use it.